### PR TITLE
#323 #121 search modal improvements

### LINKED ISSUE #121: Build 4: New customer invoice > customer searching > doesn't display all the customer list
Should have a scrollbar or something to scroll all the customer.

See image attached !
screenshot_2016-07-16-23-43-05

### LINKED ISSUE #323: SelectModal should show the autocomplete text field higher in the modal
Build Number: 19

Description: 
Currently you can only see a few rows between the text field and the keyboard. There is space above to comfortably fit a 2-3 more rows if the field was higher.

Reproducible: Always.

Reproduction Steps: Press new item in customer invoice. Look at the modal. Only observable on tablet.

Comments: Could be debatable. When the keyboard isn't there, it may look odd.

### PR DESCRIPTION
**Pull request by Chris-Petty**
Fixes #323 
Fixes #121 

Scroll works on the list.

Looks more or less as follows (Before I removed a bit of the horizontal padding):

screenshot_20180207-171329

Should test that it works on multiple, different devices. Across different android versions.

### CODE REVIEW

File: src/widgets/modals/PageContentModal.js [hunk 1]
```diff
@@ -38,41 +27,28 @@ export class PageContentModal extends React.Component {
...
-      >
-        {titleComponent}
-        <TouchableOpacity onPress={onClose} style={localStyles.closeButton}>
-          <Icon name="md-close" style={localStyles.closeIcon} />
-        </TouchableOpacity>
-        <View style={localStyles.childrenContainer}>
-          {this.props.children}
+      <BottomModal {...modalProps} style={[localStyles.modal, style]}>
+        <View style={localStyles.titleContainer}>
+          <View style={{ flex: 1 }} />
```
Comment (andreievg) [inline src/widgets/modals/PageContentModal.js]: I really like how const titleComponent was used, can we refactor this one please

**Comment by andreievg** [comment]
The code looks good, but: 

screen shot 2018-02-13 at 10 39 07 am
screen shot 2018-02-13 at 10 39 16 am

All works fine after going in and out of the app though.

**Comment by andreievg** [comment]
also:
screen shot 2018-02-13 at 10 44 05 am

**Comment by andreievg** [comment]
Tested with a production build, all work fine apart from login modal

**Comment by Chris-Petty** [comment]
What's exactly wrong? In my testing you could log in okay. There has been an issue where the login modal wouldn't go away, just eternally saying "Logging in", though.

**Comment by andreievg** [comment]
Are you testing on device? Try entering invalid username and password. You should see a screen as per above.. this is the first screen that the user sees, so should be glitch free

**Comment by Chris-Petty** [comment]
Oh sorry yea, I remember what that's depicting now: The black bar at the top is because the modal gets squished and when keyboard dismisses it doesn't grow 😕

**Comment by Chris-Petty** [comment]
I actually fixed this @andreievg, classic flexbox

**Comment by andreievg** [comment]
@Chris-Petty, happy to merge this.

**Comment by Chris-Petty** [comment]
I'll have to do a little today to make sure it's utilised in the new stocktake opening check modal, will bump then.

**Comment by Chris-Petty** [comment]
@andreievg Didn't change much, should be good to go

**Pull request merged**